tutorials/clinical/20_seeg.py

```diff
@@ -63,15 +63,37 @@
 epochs = epochs['Response'][0]  # just process one epoch of data for speed
 
 # %%
-# Let use the Talairach transform computed in the Freesurfer recon-all
-# to apply the Freesurfer surface RAS ('mri') to MNI ('mni_tal') transform.
-
-montage = epochs.get_montage()
+# First, let's plot the sensors on the brain anatomy for the individual both
+# where the contacts were implanted without modification and projecting the
+# contacts onto the inflated brain.
 
 # first we need a head to mri transform since the data is stored in "head"
 # coordinates, let's load the mri to head transform and invert it
 this_subject_dir = misc_path / 'seeg'
 head_mri_t = mne.coreg.estimate_head_mri_t('sample_seeg', this_subject_dir)
+
+view_kwargs = dict(azimuth=120, elevation=100, focalpoint=(0, 0, -15))
+brain = mne.viz.Brain('sample_seeg', subjects_dir=this_subject_dir,
+                      cortex='low_contrast', alpha=0.2, background='white')
+brain.add_sensors(epochs.info, trans=head_mri_t)
+brain.show_view(distance=350, **view_kwargs)
+
+# now, let's project onto the inflated brain surface for visualization
+proj_info = mne.preprocessing.ieeg.project_sensors_onto_inflated(
+    epochs.info, head_mri_t, 'sample_seeg', subjects_dir=this_subject_dir)
+
+brain = mne.viz.Brain('sample_seeg', subjects_dir=this_subject_dir,
+                      surf='inflated', background='white')
+brain.add_sensors(proj_info, trans=head_mri_t)
+brain.show_view(distance=500, **view_kwargs)
```

Comment:
Looks like you did not update this yet. This would just become something like the following, right?
```suggestion
# now, let's plot on the inflated brain, too
brain = mne.viz.Brain('sample_seeg', subjects_dir=this_subject_dir,
                      surf='inflated', background='white')
brain.add_sensors(epochs.info)  # automagically projects
brain.show_view(distance=500, **view_kwargs)
```